qiskit/circuit/__init__.py

```diff
@@ -562,11 +562,15 @@
 
 Hardware can be instructed to apply a real-time idle period on a given qubit.  A scheduled circuit
 (see :mod:`qiskit.transpiler`) will include all the idle times on qubits explicitly in terms of this
-:class:`Delay`.
+:class:`Delay`.  :class:`.BoxOp` can also have an explicit delay attached.
```

Comment:
Do you want to mention:
```
:attr:`.BoxOp.duration`
```
here? It feels like you're referring to `Delay` the object instead of the duration attribute in this context.